integration/exec.go

```diff
@@ -39,9 +39,7 @@ func poetExecutablePath(baseDir string) (string, error) {
 		outputPath += ".exe"
 	}
 
-	cmd := exec.Command(
-		"go", "build", "-o", outputPath, "github.com/spacemeshos/poet",
-	)
+	cmd := exec.Command("go", "build", "-o", outputPath, "github.com/spacemeshos/poet") //#nosec G204
```

Comment:
The `G204` is about executing `exec.Command` with a variable. It literally says:
>G204: Subprocess launched with variable

The change you propose doesn't fix it. G204 is intended for code audits and can be disabled after the reported code is checked. Please see the discussion in https://github.com/securego/gosec/issues/292.

Reply:
Not happy how gosec handles `G204`. This now means if we introduce a potential error where outputPath can be compromised it won't even warn us about it any more due to the suppressed warning.

But since this is the intended way how to handle it, I guess we have to leave it like this.